### PR TITLE
Try fully automating courtesy push

### PR DESCRIPTION
**Pull request by damccorm**
**Task name**: N/A CI changes

**Description**: This change should take the courtesy push the rest of the way to fully automated. With this change, we can change it to be triggered every Thursday, and it will automatically post to slack to get approval on the pr.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

### CODE REVIEW

File: ci/courtesy-push.yml [hunk 1]
```diff
@@ -44,6 +49,12 @@ steps:
     cd ..\azure-pipelines-tasks\ci\courtesy-push
     npm install
     node open-courtesy-push-pull-request.js $releaseBranch
-  displayName: Create PR in Azure DevOps
+
+    # Send message to slack channel
+    $body = '{"text": "Created tasks courtesy push PR. Someone please approve/merge this :please-puss-in-boots: ' + $env:PrLink + '"}'
+    Invoke-RestMethod -Uri $slackUri -Method Post -Body $body -ContentType 'application/json'
```
Comment (alex-peck) [inline ci/courtesy-push.yml:55]: Where is $slackUri defined?

**Comment by alex-peck** [comment]
Is there already a cron schedule in the YAML?

File: ci/courtesy-push.yml (hunk 1 above)
Comment (damccorm) [reply ci/courtesy-push.yml:55]: Its a secret in the pipeline definition

**Comment by damccorm** [comment]
> Is there already a cron schedule in the YAML?

No - we probably actually don't want it in the yaml, I was planning on overriding the courtesy push pipeline's triggers. Regardless, we shouldn't add it until this one is in

File: ci/courtesy-push.yml (hunk 1 above)
Comment (damccorm) [reply ci/courtesy-push.yml:55]: It does work - so far I've just been sending things to #damccorm-playground though

**Review by alex-peck — APPROVED** [review]
LGTM

**Pull request merged**